packages/@aws-cdk/aws-iotevents/test/integ.detector-model.ts

```diff
@@ -10,38 +18,46 @@ class TestStack extends cdk.Stack {
       attributeJsonPaths: ['payload.deviceId', 'payload.temperature'],
     });
 
+    const inputted = iotevents.Expression.currentInput(input);
+    const temperatureAttr = iotevents.Expression.inputAttribute(input, 'payload.temperature');
+    const temperatureEqual = (temperature: string) => iotevents.Expression.eq(
+      temperatureAttr,
+      iotevents.Expression.fromString(temperature),
+    );
```

Comment:
Can we actually omit these helpers? I think they cause more confusion than the code they save.